include/udx.h

```diff
@@ -83,6 +83,7 @@ typedef void (*udx_socket_send_cb)(udx_socket_send_t *req, int status);
 typedef void (*udx_socket_recv_cb)(udx_socket_t *handle, ssize_t read_len, const uv_buf_t *buf, const struct sockaddr *from);
 typedef void (*udx_socket_close_cb)(udx_socket_t *handle);
 
+typedef int (*udx_stream_firewall_cb)(udx_stream_t *handle, const struct sockaddr *from);
```

Comment:
Do we need the socket as well?

Reply:
yes! handle, socket, from?

Reply:
I'd name it `socket` as it's a stream callback.

Reply:
ah haha, i meant 3 args total in that order yea? (handle, socket, from) 🤣

Reply:
Oh, yes, that's perfect 😂